### ISSUE TITLE
Railway up fails with "x is a directory"

### ISSUE THREAD

**Issue by joshnuss** [railwayapp/cli#258]
After upgrading to the latest `@railway/cli` (2.0.7) via npm, running `railway up` resulted in an error `read <path> is a directory`:

Screenshot:
![image](https://user-images.githubusercontent.com/4437/183228106-581e26a6-41e2-4ec0-9939-304d307c41f5.png)

I then downgraded to `@railway/cli@1.8.4` and it worked fine.

**Comment by coffee-cup** [comment]
Do you have any symlinks in that directory?

**Comment by joshnuss** [comment]
Checked, there are no symlinks

<details>
  <summary>Log</summary>

```bash
customer_story (main)> ls -al deps/castore/ -al
total 32
drwxrwxr-x  4 josh josh 4096 Nov  8  2021 .
drwxrwxr-x 48 josh josh 4096 Jun 20 17:16 ..
-rw-rw-r--  1 josh josh    0 Nov  8  2021 .fetch
-rw-rw-r--  1 josh josh  272 Nov  8  2021 .hex
-rw-rw-r--  1 josh josh  453 Nov  8  2021 hex_metadata.config
drwxrwxr-x  2 josh josh 4096 Nov  8  2021 lib
-rw-r--r--  1 josh josh  875 Nov  8  2021 mix.exs
drwxrwxr-x  2 josh josh 4096 Nov  8  2021 priv
-rw-r--r--  1 josh josh 1307 Nov  8  2021 README.md
```
</details>

**Comment by coffee-cup** [comment]
Can you please check whether or not this still happens. There have been a few related fixes that might have solved this.

**Comment by joshnuss** [comment]
I tried to update via `npm install -g @railway/cli`, but it gave me 2.0.7 (same version as before) and the error still happens.

**Comment by coffee-cup** [comment]
Hmm okay, I will continue to take a look. Is the repo open source so I can attempt to repro and fix?

**Comment by joshnuss** [comment]
It's a private repo, so I tried to diagnose on my end.

I compiled the project from sources and confirmed the latest version does fix the issue. I was using the npm version which is still using v2.0.7.

The directory it was erroring on was in the `.gitgnore` using an absolute path, so it makes sense that 78dd03f8eb7b4fa9ab403896953c16a01f6d206f fixed it.

P.S. I wasn't able to compile the project with golang 1.14 (the version what listed in mod.go), got an error `controller/up.go:10:2: package io/fs is not in GOROOT (/home/josh/.asdf/installs/golang/1.14.15/go/src/io/fs)` had to use 1.16 to make that go away. 

Thanks for the help!
